### PR TITLE
Updated OpenCensus and Stackdriver Monitoring dependencies.

### PR DESCRIPTION
**Pull request by jkohen**
This pulled in a bunch of indirect dependencies.

### CODE REVIEW

File: cmd/stackdriver-prometheus-sidecar/main.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func init() {
 		panic(err)
 	}
 	if err := view.Register(
-		ocgrpc.DefaultClientViews...
+		ocgrpc.DefaultClientViews...,
```
Comment (igorpeshansky) [inline cmd/stackdriver-prometheus-sidecar/main.go]: Just curious, how is this change related to the rest of this PR?

**Comment by jkohen** [comment]
@igorpeshansky ready for review. I didn't get `go mod tidy` to pass without more effort. Next time.
